src/main/java/eu/dissco/core/translator/terms/specimen/HasMedia.java

```diff
@@ -0,0 +1,39 @@
+package eu.dissco.core.translator.terms.specimen;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import eu.dissco.core.translator.terms.Term;
+import java.util.List;
+import org.gbif.dwc.ArchiveFile;
+import org.gbif.dwc.record.Record;
+
+public class HasMedia extends Term {
+
+  public static final String TERM = ODS_PREFIX + "hasMedia";
+  private final List<String> dwcaTerms = List.of("dwc:associatedMedia");
+  private final List<String> abcdTerms = List.of("abcd:multiMediaObjects/multiMediaObject/0/fileURI");
+
+  @Override
+  public String retrieveFromDWCA(ArchiveFile archiveFile, Record rec) {
+    var result = super.searchDWCAForTerm(archiveFile, rec, dwcaTerms);
+    return toBoolean(result);
+  }
+
+  @Override
+  public String retrieveFromABCD(JsonNode unit) {
+    var result = super.searchAbcdForTerm(unit, abcdTerms);
+    return toBoolean(result);
+  }
+
+  private static String toBoolean(String result) {
```

Comment:
Mapping to a String with a boolean for a couple reasons:
- Everything is String, builds on top of generic functionality
- Might change from a boolean value to a list of the identifiers (PIDs are not yet there on translation)
Not sure though if I should have it filled in when it is there and `null` when it is false, which might be more in line with the rest and make MIDS calculation easier.

Reply:
Why not just have `String.valueOf(result==null)`